### PR TITLE
SFB-235: add shape + test for valid phone number constraint

### PR DESCRIPTION
**Pull request by JonasVanHoof**
## ID
 [SFB-235](https://binnenland.atlassian.net/browse/SFB-235)

 ## Description

A shape for validation "Valid phone number" was not yet added to the list off shapes. With this PR we add this shape to the list so next time someone uses this validation wrong we get feedback on that.

 ## Type of change

 - [ ] Bug fix
 - [x] New feature
 - [ ] Breaking change
 - [ ] Other

 ## How to test

1. run the test of the exact value constraint
2. Add the validation to your field and try to make break it  by filling in data that is not supposed to be there

 ## Links to other PR's

 - /

### CODE REVIEW

**Review by ZijneDwaasheid — COMMENTED** [review]
I would split up the min max count from the sh:in, so we can give it a different message. So when accidentally added two it gives the right message

File: public/SHACL/V2/valid-phone-number-constraint.ttl [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      sh:severity sh:Warning;
+    ],
+    [
+      sh:path form:defaultCountry ;
+      sh:datatype xsd:string ;
+      sh:minCount 1 ;
+      sh:maxCount 1 ;
+      sh:in (
+        "BE"
+        "NL"
```
Comment (ZijneDwaasheid) [inline public/SHACL/V2/valid-phone-number-constraint.ttl:30]: over here

**Comment by JonasVanHoof** [comment]
> I would split up the min max count from the sh:in, so we can give it a different message. So when accidentally added two it gives the right message

Your comments are very hard to decipher. Add the comments to a line of code please ah you just did XD thanks!

File: public/SHACL/V2/valid-phone-number-constraint.ttl (hunk 1 above)
Comment (JonasVanHoof) [reply public/SHACL/V2/valid-phone-number-constraint.ttl:30]: In who's mind is adding two country codes to a phonenumber a thing  XD

I'll add it

Comment (ZijneDwaasheid) [reply public/SHACL/V2/valid-phone-number-constraint.ttl:30]: yeah mistakes are often not intentionally

**Review by ZijneDwaasheid — APPROVED** [review]

**Pull request merged**